applications/fishing-map/src/features/map/popups/EnvironmentLayers.tsx

```diff
@@ -32,7 +32,7 @@ function EnvironmentTooltipSection({
               <div className={styles.row}>
                 <span className={styles.rowText}>
                   {parseEnvironmentalValue(feature.value)}{' '}
-                  {feature.unit && <span>{feature.unit}</span>}
+                  {feature.temporalgrid?.unit && <span>{feature.temporalgrid?.unit}</span>}
```

Comment:
There are environmental layers which doesn't contain the temporalgrid property but they have unit, so I'd use feature.unit as a fallback of temporalgrid?.unit